cmd/fs-v1.go

```diff
@@ -232,76 +412,64 @@ func (fs fsObjects) GetObject(bucket, object string, offset int64, length int64,
 	}
 
 	// Stat the file to get file size.
-	fi, err := fs.storage.StatFile(bucket, object)
+	fi, err := os.Stat(preparePath(pathJoin(fs.fsPath, bucket, object)))
```

Comment:
There seems to be a race here.

If we go via `os.Stat` here to find the size of the file without using the fd from fs.readersPool, it looks like we may read the info of a different file because a racing putobject might change the file before we go to reading the file here.